src/app/clusters/level-control/level-control.cpp

```diff
@@ -94,6 +94,12 @@ static bool areStartUpLevelControlServerAttributesNonVolatile(EndpointId endpoin
 static constexpr size_t kLevelControlStateTableSize =
     EMBER_AF_LEVEL_CONTROL_CLUSTER_SERVER_ENDPOINT_COUNT + CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT;
 
+typedef struct
```

Comment:
```suggestion
struct NextWaitTimeState
```

Reply:
I followed the style of the structure below

Reply:
That was from the original imported C code, so it's written like C.  And we never got a chance to really update it to look like C++.  But new code should be C++.

Reply:
Got it, thanks. Done